cdbot/cogs/maths.py

```diff
@@ -143,31 +142,57 @@ async def challenge(self, ctx: Context, number: int = 1):
         return await ctx.send(embed=embed)
 
     @command()
-    async def latex(self, ctx: Context, expression: str):
-        """Render a LaTeX expression."""
+    async def latex(self, ctx: Context, *, expression: str):
+        """
+        Render a LaTeX expression with https://quicklatex.com/
+        """
+        await self.latexRender(ctx, expression)
+
+    async def latexRender(self, ctx: Context, expression: str):
```

Comment:
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables

Reply:
Also, why not have the body of `latexRender` in the `latex` method instead?

Reply:
The issue is that, in order to get the entire text of the user's message rather than have discord split it up into arguments, we need to include the `*` parameter in `latex` -- so it felt easier to have this call a shared rendering function